_data/docs.yml

```
@@ -167,6 +167,18 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  rapidspdf:
+    name: RAPIDSMPF
+    path: rapidsmpf
+    desc: 'RAPIDS Multi-Process Foundation (rapidsmp) a collection of multi-gpu, distributed memory algorithms written in C++ and exposed to Python'
+    ghlink: https://github.com/rapidsai/rmm
+    cllink: https://github.com/rapidsai/rmm/blob/main/CHANGELOG.md
```

We don't have a changelog yet. But if we put it in the same spot:

```suggestion
    cllink: https://github.com/rapidsai/rapidsmpf/blob/main/CHANGELOG.md
```